tests/BaGet.Core.Tests/BaGet.Core.Tests.csproj

```diff
@@ -12,6 +12,10 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="coverlet.collector" Version="1.2.0">
```

Comment:
Please add an MSBuild variable named `CoverletCollectorPackageVersion` [here](https://github.com/loic-sharma/BaGet/blob/00466c3b764af7a0f7210716654e32f38f76fcde/tests/Directory.Build.props#L14) so that we can keep the version consistent across projects easily.